frontend/src/components/MakerForm/SelectCoordinator.tsx

```diff
@@ -83,7 +79,8 @@ const SelectCoordinator: React.FC<SelectCoordinatorProps> = ({
             >
               <Grid item>
                 <RobotAvatar
-                  shortAlias={coordinatorAlias}
+                  shortAlias={coordinator?.federated ? coordinator.shortAlias : undefined}
+                  hashId={coordinator?.federated ? undefined : coordinator.shortAlias}
```

Comment:
And here

Reply:
everything changed, thank you for noticing!